helm/policies-common/tests/abs/Pipfile.lock

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Is it the design of `abs` that every repo under test has to carry a Pipfile defining `abs` resources as well? Or is this only for the tests in this repo? At some point there will be version conflicts

Reply:
I honestly don't know. I tried to make the pipfile work with python 3.9 and then definitely broke `abs`. 

I can't tell for sure how they are connected though (mostly because of my lack of python knowledge).

Reply:
Using python 3.9 did not work because the abs container is using python 3.8.6. 

I think the Pipfile is required because `abs` creates an environment (pipenv?) for the tests with dependencies. These can differ from project to project. (just as an explanation)